src/main/scala/org/broadinstitute/dsde/firecloud/webservice/RegisterApiService.scala

```diff
@@ -51,10 +51,19 @@ trait RegisterApiService extends FireCloudDirectives with RequestBuilding with S
       }
     } ~
     pathPrefix("register") {
-      path("user" / "v1" / "termsofservice") {
-        post {
-          requireUserInfo() { _ =>
-            passthrough(samAcceptTosUrl, POST)
+      pathPrefix("user" / "v1" / "termsofservice") {
```

Comment:
Do we need to get a test around this in `RegisterApiServiceSpec`?

Reply:
I ended up modifying the integration test I had already written to use this endpoint to check whether a user has accepted the ToS with this endpoint rather than loading the user in Sam